app/models/miq_region.rb

```diff
@@ -281,6 +281,12 @@ def auth_key_configured?
     authentication_token(AUTHENTICATION_TYPE).present?
   end
 
+  def remove_auth_config
```

Comment:
I think `remove_auth_key` would be better to match `generate_auth_key`. And could we move this method to just below the generate one?